docs/getting-started/install-endpoint.asciidoc

```diff
@@ -0,0 +1,72 @@
+[role="screenshot"]
+image::images/install-endpoint/security-integration.png[]
++
+3. Select a configuration for the Elastic Agent. You can use either the **Default config**, which works on most hosts, or adds security integration to a custom or existing configuration. For more details on Elastic Agent configuration settings, see {ingest-guide}/elastic-agent-configuration.html[Configuration settings].
+4. Configure the Elastic Endpoint Security integration with a name and options description. When done configuring, select **Save integration**. Kibana redirects you back to the administration section of the security app.
++
+[role="screenshot"]
+image::images/install-endpoint/add-elastic-endpoint-security.png[]
++
+5. On the Enable Elastic Endpoint Security on your Agent's page, select the name of your new integration. To enroll your Agents with Endpoint Security, select **Enroll Agent**.
+6. Kibana redirects you back to Ingest manager to add the Elastic Agent to your host.
+
+[float]
+== Configure and enroll Elastic Agent
+
+There are two methods to install the Elastic Agent with the Elastic Endpoint Security integration using Ingest Manager.
+
+=== Enroll with Fleet
+
+1. Go to Ingest Manager. Select **Overview** > **Add agent**.
++
+[role="screenshot"]
+image::images/install-endpoint/add-agent.png[]
++
+2. In the Add agent pane of the Configurations section, download the Elastic Agent on your host's machine.
+3. After the download is complete, select an agent configuration. The selected configurations should include both **System** and **Elastic Endpoint Security** and any other integrations you chose to include when configuring an integration.
++
+[role="screenshot"]
+image::images/install-endpoint/endpoint-configuration.png[]
++
+4. After the Elastic Agent is installed on your host machine, open a command-line interface, and navigate to your Agent's directory. Copy the commands from Ingest Manager for your OS to enroll and run the Agent.
+
+=== Standalone mode
+
+If you want to manage the Elastic Agent without using Fleet to manage your hosts, you can enroll the Agent on your host in Standalone mode.
+
+1. In the Add agent pane, select the **Standalone mode** tab.
+2. Download the Elastic Agent on your host's machine and select an agent configuration.
+3. In the installed Agent's directory, create a file named `elastic-agent.yml`. Copy your configuration into the file from Step 3 of the Add agent pane. Remember to modify `ES_USERNAME` and `ES_PASSWORD` in the `outputs` section with your Elastic username and password.
+4. Save the file and start the Agent using `./elastic-agent run` command.
+5. Once the Agent begins to send data, verify that the data is viewable in Kibana by going to **Ingest Manager** > **Datasets**.
+
+[role="screenshot"]
+image::images/install-endpoint/datasets.png[]
+
+After you have enrolled the Elastic Agent on your host, select **Continue**. The host now appears on the Hosts view page inside the Elastic Security app.
```

Comment:
This isn't true for the stand-alone case, it feels a little out of place to have it here, maybe we could move it to the Fleet mode at the end, before the stand-alone case?

Reply:
Moved it to end of fleet section